roles/ceph-nfs/templates/ganesha.conf.j2

```diff
@@ -46,6 +58,8 @@ EXPORT
 
 	SecType = sys,krb5,krb5i,krb5p;
 
+	Attr_Expiration_Time = 0;
```

Comment:
Should this also be addded to rados gw export but protected by `ceph_nfs_caching_off`?

Reply:
I didn't want to protect 'Attr_Expiration_Time = 0' setting. Ideally, it should've been a EXPORT_DEFAULT option, to be effective for all exports when 'cephfs_nfs_caching_off = True'. But found that with at least NFS-Ganesha v2.5.5, 
'Attr_Expiration_Time' is only recognized as a per export option and not a EXPORT_DEFAULTS option.

I'm not sure about the caching model with Ganesha over RGW.  Also, I don't want to change the default export block settings (since 'nfs_file_gw' is true by default) unless we're certain.